website/docs/components/tooltip/partials/code/how-to-use.md

```diff
@@ -0,0 +1,98 @@
+
+* if the tooltip needs to be applied to a non-interactive element, the `TooltipButton` component should be used as it wraps the opener element with a `button` element.
+* if the tooltip needs to be applied to an element which is already interactive, the `hds-tooltip` mmodifier should be used.
+
+Currently, the tooltip uses [Tippy.js](https://atomiks.github.io/tippyjs/) under the hood for its main functionality.
+
+<!-- use the same heading order from Guidelines -->
+### Content
+
+#### Icon
+```handlebars
+  <Hds::TooltipButton @text="Here is more information">
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Inline with text
+```handlebars
+  <p>
+    <strong>Header text</strong>
+    <Hds::TooltipButton @text="Here is more information">
+      <FlightIcon @name="alert-circle" aria-label="More information" />
+    </Hds::TooltipButton>
+  </p>
+```
+
+#### Placement
+
+The tooltip appears at the “top” centered above the opener button content by default. If the tooltip is near any of the edges of the screen, its position will automatically adjust to prevent the tooltip content from being cut off so it remains readable.
+
+Use the `@placement` argument if you would like to use a different starting position for the tooltip vs. the default.
+
+
+```handlebars
+  <Hds::TooltipButton @text="Here is more information" @placement="bottom">
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Offset
+
+You can change the offset of the tooltip in relation to the opener element content if needed.
+
+
+```handlebars
+  <Hds::TooltipButton @text="Here is more information" @offset={{array 50 30}}>
+    <FlightIcon @name="alert-circle" aria-label="More information" />
+  </Hds::TooltipButton>
+```
+
+#### Extra Tippy Options
+
+Using `@extraTippyOptions` to enable rich tooltip text.
+
+Be sure to sanitize your data if enabling this option. To maintain accessibility, do not include interactive content such as links or buttons. We recommend using only basic inline-level text formatting tags such as `strong` or `em`. Using block-level tags such as `div` or `p` will make the HTML syntax invalid.
+
+```handlebars
+  <Hds::TooltipButton 
+    @extraTippyOptions={{hash allowHTML=true}}
+    @text="<b>Hello</b> <em>there</em>! Here is more info."
+  >
+    More information
+  </Hds::TooltipButton>
+```
+
+### Ember modifier
+
+An Ember modifier is available if your use case requires attaching a tooltip to an element other than a button. To be accessible, tooltips should only be attached to interactive elements. You will need to add your own styling for the element you are attaching the toolip to.
+
+#### Modifier used on a link
+
+```handlebars
+  <a href="#" {{hds-tooltip "This link takes you to more information"}}>More information</a>
```

Comment:
also for the other examples, so the options are visible without scrolling? otherwise all the code snippets look exactly the same (unless one scrolls horizontally the code snippet)

```suggestion
  <a href="#" {{hds-tooltip "Hello"}}>More information</a>
```